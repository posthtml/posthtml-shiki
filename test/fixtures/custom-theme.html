<shiki theme="my-theme">
  <h1 class="text-xl">Hello</h1>
</shiki>
