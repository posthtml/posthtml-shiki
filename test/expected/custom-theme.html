<pre class="shiki my-theme" style="background-color:#1e1e1e;color:#bbbbbb" tabindex="0"><code><span class="line"><span style="color:#BBBBBB">&#x3C;h1 class=</span><span style="color:#888">"text-xl"</span><span style="color:#BBBBBB">>Hello&#x3C;/h1></span></span></code></pre>
